hash_chunker/hash_chunker.py

```diff
@@ -99,3 +103,19 @@ def _position_to_hex(self, position: int) -> str:
             hexed = self.hex_zero * zeros_count + hexed
         hexed += self.hex_zero * (self.hash_max_length - len(hexed))
         return hexed[: self.chunk_hash_length]
+
+    def _exceptions_(self, arg):
+        if type(arg) == type(True):
+            raise TypeError("arguments should be int")
+        if not isinstance(arg, int):
+            raise TypeError("arguments should be int")
```

Comment:
Update text to starting from capital letter, and with period and set arg name to message.